docs/core/install/remove-runtime-sdk-versions.md

```diff
@@ -129,18 +152,18 @@ Starting with Visual Studio 2019 version 16.3, Visual Studio is in charge of its
 
 ::: zone-end
 
-## Remove the NuGet fallback folder
+## Remove the NuGet fallback directory
 
-Before .NET Core 3.0 SDK, the .NET Core SDK installers used a folder named *NuGetFallbackFolder* to store a cache of NuGet packages. This cache was used during operations such as `dotnet restore` or `dotnet build /t:Restore`. The *NuGetFallbackFolder* is located at *C:\Program Files\dotnet\sdk* on Windows and at */usr/local/share/dotnet/sdk* on macOS.
+Before .NET Core 3.0 SDK, the .NET Core SDK installers used a directory named *NuGetFallbackFolder* to store a cache of NuGet packages. This cache was used during operations such as `dotnet restore` or `dotnet build /t:Restore`. The *NuGetFallbackFolder* is located at *C:\Program Files\dotnet\sdk* on Windows and at */usr/local/share/dotnet/sdk* on macOS.
```

Comment:
Does the fallback directory also apply to Linux? If so, would it be the same as macOS?

Reply:
It may be. There is a lot of variation as to where this folder could be located depending on the different ways .NET is installed. I'm clarifying this a little bit to be more generic as to its location.